mappings/InventoryID-Bibtex.csv

```diff
@@ -1,3844 +1,3844 @@
-568,Kelley1963,upsid,http://web.phonetik.uni-frankfurt.de/L/L2902.html
-568,Kostic_etal1977,upsid,http://web.phonetik.uni-frankfurt.de/L/L2902.html
-568,Krishnamurti1961,upsid,http://web.phonetik.uni-frankfurt.de/L/L2902.html
-568,Lisker1963,upsid,http://web.phonetik.uni-frankfurt.de/L/L2902.html
-568,Sastry1972,upsid,http://web.phonetik.uni-frankfurt.de/L/L2902.html
-569,Dalby1966,upsid,http://web.phonetik.uni-frankfurt.de/L/L4109.html
-569,Wilson1961,upsid,http://web.phonetik.uni-frankfurt.de/L/L4109.html
-570,TuckerBryan1966,upsid,http://web.phonetik.uni-frankfurt.de/L/L4211.html
-571,Morris1984,upsid,http://web.phonetik.uni-frankfurt.de/L/L2437.html
-572,Beaumont1979,upsid,http://web.phonetik.uni-frankfurt.de/L/L2425.html
-573,Bloomfield1917,upsid,http://web.phonetik.uni-frankfurt.de/L/L2414.html
-573,SchachterOtanes1972,upsid,http://web.phonetik.uni-frankfurt.de/L/L2414.html
-574,Abramson1962,upsid,http://web.phonetik.uni-frankfurt.de/L/L2400.html
-574,Haas1956,upsid,http://web.phonetik.uni-frankfurt.de/L/L2400.html
-574,Haas1964,upsid,http://web.phonetik.uni-frankfurt.de/L/L2400.html
-574,Kruatrachue1960,upsid,http://web.phonetik.uni-frankfurt.de/L/L2400.html
-574,Noss1954,upsid,http://web.phonetik.uni-frankfurt.de/L/L2400.html
-574,Noss1964,upsid,http://web.phonetik.uni-frankfurt.de/L/L2400.html
-575,Prasse1972,upsid,http://web.phonetik.uni-frankfurt.de/L/L4357.html
-576,Klingenheben1966,upsid,http://web.phonetik.uni-frankfurt.de/L/L4351.html
-576,Palmer1962,upsid,http://web.phonetik.uni-frankfurt.de/L/L4351.html
-577,Osborne1974,upsid,http://web.phonetik.uni-frankfurt.de/L/L8351.html
-577,Lee1983,upsid,http://web.phonetik.uni-frankfurt.de/L/L8351.html
-577,Lee1984,upsid,http://web.phonetik.uni-frankfurt.de/L/L8351.html
-578,Post1966,upsid,http://web.phonetik.uni-frankfurt.de/L/L2431.html
-578,Schlegel1971,upsid,http://web.phonetik.uni-frankfurt.de/L/L2431.html
-579,StoryNaish1973,upsid,http://web.phonetik.uni-frankfurt.de/L/L6102.html
-579,Swanton1909,upsid,http://web.phonetik.uni-frankfurt.de/L/L6102.html
-579,Swanton1911,upsid,http://web.phonetik.uni-frankfurt.de/L/L6102.html
-580,Schadeberg1981b,upsid,http://web.phonetik.uni-frankfurt.de/L/L4142.html
-581,TuckerBryan1966,upsid,http://web.phonetik.uni-frankfurt.de/L/L4210.html
-582,Voorhoeve1965,upsid,http://web.phonetik.uni-frankfurt.de/L/L8601.html
-583,Key1968,upsid,http://web.phonetik.uni-frankfurt.de/L/L6812.html
-583,Van-Wynen1962,upsid,http://web.phonetik.uni-frankfurt.de/L/L6812.html
-584,Lynch1978,upsid,http://web.phonetik.uni-frankfurt.de/L/L2426.html
-585,Aschmann1946,upsid,http://web.phonetik.uni-frankfurt.de/L/L6713.html
-586,Bergman_etal1969,upsid,http://web.phonetik.uni-frankfurt.de/L/L4112.html
-587,Suarez1983,upsid,http://web.phonetik.uni-frankfurt.de/L/L6772.html
-588,Trumai1975,upsid,http://web.phonetik.uni-frankfurt.de/L/L6865.html
-589,Brown1973,upsid,http://web.phonetik.uni-frankfurt.de/L/L8623.html
-590,Hoijer1946,upsid,http://web.p
```

Comment:
what's going on with this line?

Reply:
(be sure to click through to the file...  the preview of which lines I'm commenting on is messed up.  The ones I'm asking about are inventories 2409, 2436, and 2505 --- line numbers 3217, 3246, and 3319)